### PR TITLE
LPD-27029 | Master

### PR DESCRIPTION
**Pull request by interaminense**

### CODE REVIEW

**Comment by interaminense** [comment]
ci:test:sf

**Comment by interaminense** [comment]
ci:test:relevant

**Comment by interaminense** [comment]
ci:test:analytics-cloud-acceptance

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud-acceptance

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/workspaces/AddWorkspaceForm.tsx [hunk 1]
```diff
@@ -55,10 +58,8 @@ const getProjectLocations = (): {label: string; value: string}[] => {
 
 const getDefaultServerLocation = (): string => {
 	switch (FARO_ENV) {
-		case 'dev':
-			return DEV;
-		case 'uat':
-			return UAT;
+		case FaroEnv.Staging:
+			return INTERNAL;
```
Comment (marcellustavares) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/workspaces/AddWorkspaceForm.tsx:62]: @interaminense `INTERNAL` is not the staging env.

**Review by marcellustavares — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/workspaces/AddWorkspaceForm.tsx (hunk 1 above)
Comment (interaminense) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/workspaces/AddWorkspaceForm.tsx:62]: Which server location should appear to the env that is using stating?

**Comment by interaminense** [comment]
ci:test:sf

**Comment by interaminense** [comment]
ci:test:relevant

**Comment by interaminense** [comment]
ci:test:relevant

**Comment by interaminense** [comment]
ci:test:relevant

**Comment by interaminense** [comment]
ci:forward:force

**Pull request closed without merging**